akka-actor/src/main/scala/akka/io/dns/DnsResourceRecords.scala

```diff
@@ -14,7 +14,7 @@ import akka.util.{ ByteIterator, ByteString, ByteStringBuilder }
 import scala.annotation.switch
 
 @ApiMayChange
-sealed abstract class ResourceRecord(val name: String, val ttl: Int, val recType: Short, val recClass: Short)
+sealed abstract class ResourceRecord(val name: String, val ttlSeconds: Int, val recType: Short, val recClass: Short)
```

Comment:
`ttlInSeconds`

Reply:
I wanted to be consistent with `ttlMillis`in https://github.com/akka/akka/blob/b72d3090af5d29d3ad0762c1b871a18cb75d540b/akka-actor/src/main/scala/akka/io/dns/internal/AsyncDnsCache.scala#L58-L62 but I guess I can change both to the `ttlInZzzz` format.